workflow/snakemake_rules/main_workflow.smk

```diff
@@ -445,6 +445,7 @@ rule combine_samples:
         augur filter \
             --sequences {input.sequences} \
             --metadata {input.metadata} \
+            --skip-checks \
```

Comment:
> (1) is not useful since sanitize_sequences.py already drops duplicates.

Ah, took me a while to realize that the sequences _always_ run through `sanitize_sequences.py` in the `build_align` rule _after_ this filter step. 

Aside, small nit to update `combine_samples` rule's message to reflect this change.

Reply:
> Ah, took me a while to realize that the sequences _always_ run through `sanitize_sequences.py` in the `build_align` rule _after_ this filter step.

Oh whoops, I only added an explanation for one of the code paths (where `skip_sanitize_metadata` is false). In practice we have it set to true everywhere, and in that case ncov-ingest already drops duplicates ([e.g.](https://github.com/nextstrain/ncov-ingest/blob/5a41e381db0522e3da6bf97406bc9b02a608783f/bin/transform-rki-data-to-ndjson#L26)). Updated the commit message.

> small nit to update `combine_samples` rule's message to reflect this change.

I've removed the rule message in 01d378a619dbab67fe831d7ae6e9f63d801917d3.

Reply:
> (1) is not useful since inputs are already deduplicated prior to running
this rule (either by ncov-ingest or sanitize_sequences.py).

Hmm, now I'm wondering if this is true. This rule is combining the results of multiple subsampling results, so even if the inputs are deduplicated, a single record could have been selected in two different subsample groups and need to be deduplicated.

Reply:
`--skip-checks` applies on the [input directly](https://github.com/nextstrain/augur/blob/c5c1af171c15f1994d7e6c28aed7092d3b5262b5/augur/filter/_run.py#L82-L84), before `--include`. If the same id appears in multiple `--include` files, it will only appear once in the output because the force-included ids are [stored as a set](https://github.com/nextstrain/augur/blob/c5c1af171c15f1994d7e6c28aed7092d3b5262b5/augur/filter/include_exclude_rules.py#L813) which automatically deduplicates.

I know it's confusing, but let me know if it's still not clear! Good to make sure that this change does not introduce any new cracks.

Reply:
Ooh, okay that makes sense now. 

Bear with me for one more edge case: could this cause errors if a user starts with an [aligned sequence input](https://docs.nextstrain.org/projects/ncov/en/latest/reference/workflow-config-file.html#aligned) that is not produced by us that includes duplicates? `_get_unified_alignment` will directly use an `aligned` sequence input if [there is only one input](https://github.com/nextstrain/ncov/blob/137ffc479a1bfe4e2c6ae1bbcd1b22e3afc5f8dd/workflow/snakemake_rules/common.smk#L167-L168).